action.yml

```diff
@@ -20,6 +20,18 @@ inputs:
     required: false
     default: 'all'
     description: 'The type of tests to be run by the test runner.'
+  enableCodeCoverage:
+    required: false
+    default: 'false'
+    description: 'Should code coverage results be collected for the test run.'
```

Comment:
Good point, I'll change it to just `coverage` as that makes a lot more sense.

Reply:
One reason to include this as a public api is that it requires the `-debugCodeOptimization` and `-enableCodeCoverage` flags. But now that I write this down, it would make more sense to always enable coverage by default and have a separate flag to turn off the `-debugCodeOptimization` or `-enableCodeCoverage`.  Since this is for running tests, most end users probably won't mind the `-debugCodeOptimization` but if someone wanted to run something like a performance test it would help. But I think you probably would need a full build for that anyway.

Given all that reasoning, let's leave it as enabled by default, I think I like that option better! It also makes some of the other changes much simpler.